nutkit/protocol/feature.py

```diff
@@ -42,6 +44,9 @@ class Feature(Enum):
     # If there are more than records, the driver emits a warning.
     # This method is supposed to always exhaust the result stream.
     API_RESULT_SINGLE_OPTIONAL = "Feature:API:Result.SingleOptional"
+    # The session supports notification filters configuration.
+    API_SESSION_NOTIFICATION_FILTERS = \
+        "Feature:API:Session.NotificationFilters"
```

Comment:
Maybe we want this one to be 
```suggestion
        "Feature:API:Session:NotificationFilters"
```
?
Because it's not a method/method/attribute (same for `API_DRIVER_NOTIFICATION_FILTERS`).